frontend/src/app/components/network-information/network-information.component.css

```diff
@@ -19,6 +19,9 @@
 .title {
   font-size: 1rem;
   font-weight: 500;
+}
```

Comment:
When we remove this here the info box titles will not be displayed in small caps any more.

Reply:
I added a `small-caps` class and i used it on the titles that should be small caps.